scripts/artifacthub/hub.go

```diff
@@ -221,11 +224,38 @@ func addArtifactHubMetadata(sourceDirectory, destinationPath, ahBasePath string,
 	}
 }
 
-func checkVersion(artifactHubMetadata *ArtifactHubMetadata, constraintTemplate map[string]interface{}, newConstraintTemplateHash string) error {
-	// compare hash
-	if artifactHubMetadata.Digest != newConstraintTemplateHash {
+func checkVersion(artifactHubMetadata *ArtifactHubMetadata, constraintTemplate map[string]interface{}, githubSourceRelativePath string) error {
+	// compare hash with template.yaml in github
+	githubTemplateURL := sourceURL + githubSourceRelativePath
+	resp, err := http.Get(githubTemplateURL)
+	if err != nil {
+		fmt.Println("error while getting constraint template from github")
+		panic(err)
+	}
+	if resp.StatusCode == http.StatusNotFound {
+		fmt.Printf("constraint template %s not found in github. It is likely that constraint template is being updated locally and not merged to github yet.\n", githubSourceRelativePath)
+		
```

Comment:
```suggestion
```

Reply:
I usually put the `return` on the new line for better code readability. LMK if you have any opinions about the same.